ichub-backend/controllers/fastapi/app.py

```diff
@@ -114,4 +114,22 @@ async def twin_management_share_catalog_part_twin(catalog_part_twin_share: Catal
 
 @app.post("/twin-management/twin-aspect", response_model=TwinAspectRead, tags=["Twin Management"])
 async def twin_management_create_twin_aspect(twin_aspect_create: TwinAspectCreate) -> TwinAspectRead:
-    return twin_management_service.create_twin_aspect(twin_aspect_create)
\ No newline at end of file
+    return twin_management_service.create_twin_aspect(twin_aspect_create)
+
+@app.get("/submodel-dispatcher/{semantic_id}/{global_id}", response_model=Dict[str, Any], tags=["Submodel Dispatcher"])
```

Comment:
```suggestion
@app.get("/submodel-dispatcher/{semantic_id}/{global_id}/submodel", response_model=Dict[str, Any], tags=["Submodel Dispatcher"])
```

Quick question, do we need the API path to end with submodel, or does the dataplane remove that part?